src/renderer/src/helpers/UtilsHelper.ts

```diff
@@ -204,4 +204,8 @@ export class UtilsHelper {
 
     return false
   }
+
+  static removeSpecialCharacters(string: string, allowSpaces: boolean) {
```

Comment:
I wouldn't use the `string` variable name, because it's a reserved word for types. I'd use `text`.